rackcity/views/user_views.py

```diff
@@ -37,10 +37,9 @@
 from rest_framework.permissions import IsAuthenticated
 
 
-class RegisterNameView(PermissionRequiredMixin, RegisterView):
+class RegisterNameView(RegisterView):
+    permission_classes = IsAuthenticated, RegisterUserPermission
     serializer_class = RegisterNameSerializer
-    permission_required = PermissionPath.USER_WRITE.value
-    raise_exception = True
```

Comment:
the PermissionRequiredMixin was not working, so no requests to this endpoint were passing the permissions check. I have no idea why it was working for creating one user but not two in a row before our eval